cmd/okctl/create.go

```diff
@@ -45,6 +45,12 @@ func buildCreateCommand(o *okctl.Okctl) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "create",
 		Short: "Create commands",
+		Long: `Create various kinds of clusters.
```

Comment:
I'm currently adding a `okctl create application` command, which is not a cluster and will make this description misleading

Reply:
Let's see who merges first :trollface: 

OK, changed to `resources`.